### PR TITLE
Use go modules

### LINKED ISSUE #17: Use Go modules
From go 1.14 go modules are the default.

### PR DESCRIPTION
**Pull request by gildub**
Fix https://github.com/konveyor/openshift-velero-plugin/issues/17

### CODE REVIEW

**Review by dymurray — APPROVED** [review]
Tested and worked for me

File: vendor/github.com/konveyor/openshift-velero-plugin/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+Apache License
```
Comment (sseago) [inline vendor/github.com/konveyor/openshift-velero-plugin/LICENSE]: I don't think these files should be showing up under vendor. It looks like this repo is including itself as its own dependency here, putting files under vendor/ that also exist at the top level.

**Review by sseago — COMMENTED** [review]
Overall, this seems fine, other than what looks like mistakenly including this repo's own files as a dep under vendor/

File: vendor/github.com/konveyor/openshift-velero-plugin/LICENSE (hunk 1 above)
Comment (gildub) [reply vendor/github.com/konveyor/openshift-velero-plugin/LICENSE]: @sseago, you're right, that was a mistake, it's fixed now, thanks

Comment (gildub) [reply vendor/github.com/konveyor/openshift-velero-plugin/LICENSE]: @sseago as you found out there is a typo in the project path. Rebuilding it.

**Review by sseago — APPROVED** [review]
Looks good now. I think we're good to go once the CI build passes.

**Pull request merged**